pipeline/frontend/yaml/compiler/settings/params.go

```diff
@@ -26,15 +26,15 @@ import (
 
 // ParamsToEnv uses reflection to convert a map[string]interface to a list
 // of environment variables.
-func ParamsToEnv(from map[string]any, to map[string]string, prefix string, getSecretValue func(name string) (string, error)) (err error) {
+func ParamsToEnv(from map[string]any, to map[string]string, prefix string, upper bool, getSecretValue func(name string) (string, error)) (err error) {
```

Comment:
Can we call it `toUpper`? Sorry, I know a bit nitpicky.

Reply:
While I fully disagree about this way of communication, I don't care much enough to further discuss it...

Reply:
I thumb'ed it down because in Go function param naming should be short (you will rarely find anything more then few chars let a lone multiple words with uppercase characters)

Reply:
sorry if I offended it with it, was not my intention

Reply:
No worries. I would just like to learn/hear opinions, so a plain thumbs-down says nothing to me. Thanks for adding context.